build.sbt

```diff
@@ -117,7 +117,8 @@ ThisBuild / tlVersionIntroduced := Map("3" -> "2.6.1")
 
 lazy val commonJvmSettings = Seq(
   Test / fork := true,
-  Test / javaOptions := Seq("-Xmx3G")
+  Test / javaOptions := Seq("-Xmx3G"),
+  doctestGenTests := { if (tlIsScala3.value) Nil else doctestGenTests.value }
```

Comment:
Doctests are supported on Scala 3, but our examples simply don't cross-compile e.g. kind projector syntax for `FunctionK`.